oi/type_graph/RemoveMembers.cpp

```diff
@@ -54,19 +54,12 @@ void RemoveIgnored::visit(Class& c) {
     accept(child);
   }
 
-  for (size_t i = 0; i < c.members.size(); i++) {
-    if (!ignoreMember(c.name(), c.members[i].name)) {
-      continue;
-    }
-    auto& primitive = typeGraph_.makeType<Primitive>(Primitive::Kind::Int8);
-    auto& paddingArray =
-        typeGraph_.makeType<Array>(primitive, c.members[i].type().size());
-    c.members[i] =
-        Member{paddingArray, AddPadding::MemberPrefix, c.members[i].bitOffset};
-  }
+  std::erase_if(c.members, [this, &c](Member member) {
+    return ignoreMember(c.name(), member.name);
```

Comment:
Given `c.name()` returns a copy, can you gather it before and capture it by reference?

Reply:
No more copy: #264